core/internal/wbvalue/image.go

```diff
@@ -66,18 +65,26 @@ func ImageFromData(
 	}, nil
 }
 
-// HistoryValueJSON is the image metadata to keep in the run history.
-//
-// The `filePath` is the run file path to which the image was saved.
-func (img Image) HistoryValueJSON(filePath paths.RelativePath) (string, error) {
+// HistoryImageValuesJSON is the metadata for multiple images to keep in the run history.
+// The `filePaths` are the run file paths to where each image was saved.
```

Comment:
nit: there should be a blank line between the summary sentence and additional details in all doc comments